tez-dag/src/test/java/org/apache/tez/dag/app/TestDAGAppMaster.java

```diff
@@ -96,6 +105,7 @@ public class TestDAGAppMaster {
   private static final String CLASS_SUFFIX = "_CLASS";
   private static final File TEST_DIR = new File(System.getProperty("test.build.data"),
       TestDAGAppMaster.class.getName()).getAbsoluteFile();
+  private final FileSystem mockFs = mock(FileSystem.class);
```

Comment:
as this is just a single line now, it doesn't have to be a field, you can have it in the methods

Reply:
done